tools/tsp-client/src/commands.ts

```diff
@@ -485,36 +480,44 @@ export async function generateConfigFilesCommand(argv: any) {
   if (Object.keys(overrideJson).length > 0) {
     emitterPackageJson["overrides"] = overrideJson;
   }
+  
+  const emitterPath = resolveEmitterPathFromArgs(argv) ?? joinPaths(repoRoot, defaultRelativeEmitterPackageJsonPath);
```

Comment:
This command doesnt support the `emitter-package-json-path` arg it supports a `package-json` path which is meant to be used with the package.json file of the target emitter to pull deps from there. It also is meant to create the default config files (emitter-package.json and emitter-package-lock.json) that tsp-client needs to be able to run commands generally. We dont need to change this. At most we could enable an option for you to specify an alternate name to save these files to.

Reply:
Yeah I think we need the same support for specifying the emitter package path in the generate config commands. This is so that we are able to use tsp-client to regen these files after we ship the new emitter.

Reply:
So then you should revert these changes and add a flag to allow you to specify an alternate name for the files

Reply:
How long are you planning on hosting these 2 sets of emitter-package files together? (the default file and the alternate emitter config file)

Reply:
> So then you should revert these changes and add a flag to allow you to specify an alternate name for the files

In this command you'd only need to update the file name when it's being written. And yes please update the docs on this one as well

Reply:
I also updated the name of the file that is logged.